src/main/resources/l10n/JabRef_fr.properties

```diff
@@ -1757,7 +1783,7 @@ journal\ not\ found\ in\ abbreviation\ list=Journal non trouvé dans la liste d'
 Unhandled\ exception\ occurred.=Une exception non gérée est survenue.
 
 strings\ included=chaînes incluses
-Escape\ underscores=Échapper les soulignements
+Escape\ underscores=Échapper les soulignements (_)
```

Comment:
This is probably not how we want translations?

Reply:
The idea of showing the symbol came from the same menu for "&", because most French people do not know what is an "esperluette".
Just explaining.

Reply:
ok, i see. Thanks!